frontend/src/views/HomeView.vue

```diff
@@ -17,16 +31,184 @@ const { getConfig } = storeToRefs(useConfigStore());
     {{ getConfig?.notificationBanner }}
   </Message>
 
-  <h3 v-if="!getIsAuthenticated">
-    Please log in to continue.
-  </h3>
-  <h3 v-else>
-    Welcome to BCBox.
-  </h3>
+  <div class="flex flex-column mr-8 ml-8">
+    <div class="flex justify-content-center mb-5">
+      <h1 class="font-bold">
+        Store and share files in BCBox
+      </h1>
+    </div>
+    <div class="flex justify-content-center mb-5">
+      <p class="text-xl text-center">
+        This website uses the <a href="https://bcgov.github.io/common-service-showcase/services/coms.html">Common Object Management Service</a> so
+        <a href="https://www2.gov.bc.ca/gov/content/governments/services-for-government/information-management-technology/id-services/idir">IDIR</a> and <a href="https://www.bceid.ca/">BCeID</a>
+        users can <br /> upload and share files with each other.
+      </p>
+    </div>
+    <div class="flex justify-content-center mb-5">
+      <h3 v-if="!getIsAuthenticated">
+        <Button
+          @click="login()"
+        >
+          Log in to get started
+        </Button>
+      </h3>
+      <h3 v-else>
+        <Button
+          @click="goToBuckets()"
+        >
+          Go to My Buckets
+        </Button>
+      </h3>
```

Comment:
The button action for both of these should be the same: goToBuckets(). We shouldn't be permuting its behavior here because we should be depending on underlying infra to do the login for us if necessary. The only thing that should be changing here is the flavor text, and that can be a simple computed based on whether you're actually authenticated or not.

Another note is that for link navigation instances like this, we should be leveraging `<router-link>` instead. This could be collapsed down to something like the following IIRC:
```vue
<router-link to="/list/buckets">
  <Button>
    {{ !getIsAuthenticated ? 'Log in to get started' : 'Go to My Buckets' }}
  </Button>
</router-link>
```